.github/workflows/canaries_prune.yml

```diff
@@ -12,6 +12,9 @@ on:
           - linux
           - windows
         required: false
+      tag:
+        description: 'Tag to prune'
```

Comment:
I think type of the input is recommended to have

Reply:
👍  done in https://github.com/newrelic/infrastructure-agent/pull/1794/commits/3b6ea17d7d4da595b4b91257120a578eae92bf13